Makefile

```
@@ -345,4 +342,10 @@ test-aws-ou-logic: check-ou-mapping-configmap-env check-ou-mapping-configmap-env
 
 #s Test all
 .PHONY: test-all
-test-all: lint test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess test-awsfederatedrole test-aws-ou-logic 
+test-all: lint clean-operator test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess test-awsfederatedrole test-aws-ou-logic 
+
+.PHONY: clean-operator
+clean-operator: delete-accountclaim-namespace delete-ccs-namespace delete-ccs-2-namespace
+	oc delete accounts --all -n ${NAMESPACE}
+	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
+	oc delete awsfederatedrole --all -n ${NAMESPACE}
```

Rather than making the targets themselves always succeed, you could make them do so only when run under `clean-operator` via something like:

```suggestion
clean-operator:
	$(MAKE) delete-accountclaim-namespace || true
	$(MAKE) delete-ccs-namespace || true
	$(MAKE) delete-ccs-2-namespace || true
	oc delete accounts --all -n ${NAMESPACE}
	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
	oc delete awsfederatedrole --all -n ${NAMESPACE}
```

---

Thanks for the suggestion.  Added.